README.md

```diff
@@ -3,32 +3,63 @@
-* Code: [https://github.com/datastax/php-driver](https://github.com/datastax/php-driver)
-* Jira: [https://datastax-oss.atlassian.net/browse/PHP](https://datastax-oss.atlassian.net/browse/PHP)
-* Mailing List: [https://groups.google.com/a/lists.datastax.com/forum/#!forum/php-driver-user](https://groups.google.com/a/lists.datastax.com/forum/#!forum/php-driver-user)
-* IRC: #datastax-drivers on [irc.freenode.net](http://freenode.net>)
+This is a wrapper around the [DataStax C/C++ Driver for Apache Cassandra].
 
 __Note__: DataStax products do not support big-endian systems.
 
-## What's new in v1.2.0/v1.3.0:
+## Getting the Driver
+
+Binary versions of the driver are available for multiple operating systems and
+multiple versions of PHP which can be obtained from the
+[DataStax download server]. The source code is made available via [GitHub]. _If
+using [DataStax Enterprise] use the [DSE PHP driver] instead_.
 
-* Support for [`duration`](http://datastax.github.io/php-driver/api/Cassandra/class.Duration/)
-* `Session::execute()` and `Session::executeAsync()` now support a [simple string](http://datastax.github.io/php-driver/features/simple_string_queries/) for the query CQL and a simple array for the query execution option.
+## What's new in v1.2.0/v1.3.0
+
+* Support for [`duration`]
+* `Session::execute()` and `Session::executeAsync()` now support a
+  [simple string] for the query CQL and a simple array for the query execution
+  option
 * Full support for Apache Cassandra 2.2 and 3.0+
-* Support for [`tinyint` and `smallint`](http://datastax.github.io/php-driver/features/datatypes/#using-cassandra-the-tinyint-and-smallint-types)
-* Support for [`date`](http://datastax.github.io/php-driver/features/datatypes/#using-cassandra-date-type) and [`time`](http://datastax.github.io/php-driver/features/datatypes/#using-cassandra-time-type)
-* Support for [user-defined function and aggregate](http://datastax.github.io/php-driver/features/function_and_aggregate_metadata) metadata
-* Support for [secondary index](http://datastax.github.io/php-driver/features/secondary_index_metadata) and [materialize view](http://datastax.github.io/php-driver/features/function_and_aggregate_metadata) metadata
+* Support for [`tinyint` and `smallint`]
+* Support for [`date`] and [`time`]
+* Support for [user-defined function and aggregate] metadata
+* Support for [secondary index] and [materialized view] metadata
+
+## Compatibility
+
+This driver works exclusively with the Cassandra Query Language v3 (CQL3) and
+Cassandra's native protocol. The current version works with:
+
+* Apache Cassandra versions 2.1, 2.2 and 3.0+
+* PHP 5.6, PHP 7.0, and PHP 7.1
+  * 32-bit (x86) and 64-bit (x64)
+  * Thread safe (TS) and non-thread safe (NTS)
+* Compilers: GCC 4.1.2+, Clang 3.4+, and MSVC 2010/2012/2013/2015
+
+If using [DataStax Enterprise] the [DSE PHP driver] provides more features and
+better compatibility.
+
+## Documentation
+
+* [Home]
+* [API]
```

Comment:
👍 I will remove the link to features from the opening paragraph and place it into the documentation section